test/create-webpack-app/test.utils.js

```diff
@@ -1,13 +1,16 @@
-/* eslint-disable node/no-unpublished-require */
 "use strict";
 
 const testUtilsPkg = require("../utils/test-utils.js");
 const { processKill } = testUtilsPkg;
 const path = require("path");
+// eslint-disable-next-line node/no-unpublished-require
 const execa = require("execa");
 const { node: execaNode } = execa;
+// eslint-disable-next-line node/no-unpublished-require
 const stripAnsi = require("strip-ansi");
+// eslint-disable-next-line node/no-unpublished-require
 const concat = require("concat-stream");
+// eslint-disable-next-line node/no-unpublished-require
```

Comment:
Why we need it? I think it should work, or we use old version?

Reply:
We don't need it, the branch was outdated I just merged the master branch